docs/src/atomistic/outputs/energy.rst

```diff
@@ -173,10 +173,55 @@ Energy ensembles must have the following metadata:
       ``"energy"``, with entries ranging from 0 to the number of members of the
       ensemble minus one.
 
-Currently, no simulation engines can use the ``"energy_ensemble"`` output.
+The following simulation engines can use the ``"energy_ensemble"`` output:
+
+.. grid:: 1 1 1 1
+
+  .. grid-item-card::
+    :text-align: center
+    :padding: 1
+    :link: engine-ipi
+    :link-type: ref
+
+    |ipi-logo|
 
 Energy ensemble gradients
 -------------------------
 
 The gradient metadata for energy ensemble is the same as for the ``energy``
 output (see :ref:`energy-output-gradients`).
+
+Energy uncertainty
```

Comment:
You should also add a link to this (ideally with a pretty figure, I can help to make one) in the `docs/src/atomistic/outputs/index.rst`